.gitignore

```diff
@@ -59,6 +59,9 @@ venv
 *.iml
 *.iws
 
+# Visual Studio Code
+.vscode
```

Comment:
_nitpick_
I think these kinds of development environment specific ignores could go to everyone's global `.gitignore`. What do you think?

Reply:
Oh yeah, forgot about. I've updated my global one.

https://github.com/rowlando/homedir/blob/master/etc/git/ignore